vs2015/SumatraPDF.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please undo changes here. Those files are auto-generated from `premake5.lua` and will be over-written on next generation. You need to add Theme.cpp/Theme.h to list of files in premake5.files.lua and re-generated files as described at the top of premake5.lua.